OMCompiler/Compiler/BackEnd/BackendDAE.mo

```diff
@@ -783,15 +783,18 @@ type SymbolicJacobian = tuple<BackendDAE,               // symbolic equation sys
                               list<Var>,                // diff vars (independent vars)
                               list<Var>,                // diffed vars (residual vars)
                               list<Var>,                // all diffed vars (residual vars + dependent vars)
-                              list< .DAE.ComponentRef>    // original dependent variables
+                              list< .DAE.ComponentRef>  // original dependent variables
                               >;
 
+type SparsePatternCref = tuple< .DAE.ComponentRef, list< .DAE.ComponentRef>>;
```

Comment:
You could probably name this somehow: list< .DAE.ComponentRef> and then use it in SparsePaternCref and also in SparsePattern.

Reply:
You mean create a type for `list<DAE.ComponentRef>` in DAE.mo somewhere?

Reply:
That works also, you could call it CrefList or Crefs.

Reply:
Changing `list<DAE.ComponentRef>` everywhere is not easy done automatically. It is used in a lot of places and for the bootstrapped compiler. I would prever to keep it this way for now.

Reply:
Is ok. We can do that later.